docs/centralized/main.md

```diff
@@ -0,0 +1,3 @@
+# Telescope - Construction with Bounded DFS
```

Comment:
We should probably rename the directory `centralized` -> `centralized_telescope` to mirror the structure in `src/`.